src/Microsoft.AspNetCore.Diagnostics/ExceptionHandler/ExceptionHandlerMiddleware.cs

```diff
@@ -65,6 +65,7 @@ public class ExceptionHandlerMiddleware
                     var exceptionHandlerFeature = new ExceptionHandlerFeature()
                     {
                         Error = ex,
+                        Path = originalPath.Value
```

Comment:
Can add a trailing comma here... though given the other comments in this PR, this code might change anyway.